plugins/PostProcessingPlugin/scripts/ChangeAtZ.py

```diff
@@ -4,496 +4,954 @@
+					"label": "Change Bed Temp",
+					"description": "Select if Bed Temperature has to be changed",
+					"type": "bool",
+					"default_value": false
+				},
+				"h2_bedTemp": {
+					"label": "Bed Temp",
+					"description": "New Bed Temperature",
+					"unit": "C",
+					"type": "float",
+					"default_value": 60,
+					"minimum_value": "0",
+					"minimum_value_warning": "30",
+					"maximum_value_warning": "120",
+					"enabled": "h1_Change_bedTemp"
+				},
+				"i1_Change_extruderOne": {
+					"label": "Change Extruder 1 Temp",
+					"description": "Select if First Extruder Temperature has to be changed",
+					"type": "bool",
+					"default_value": false
+				},
+				"i2_extruderOne": {
+					"label": "Extruder 1 Temp",
+					"description": "New First Extruder Temperature",
+					"unit": "C",
+					"type": "float",
+					"default_value": 190,
+					"minimum_value": "0",
+					"minimum_value_warning": "160",
+					"maximum_value_warning": "250",
+					"enabled": "i1_Change_extruderOne"
+				},
+				"i3_Change_extruderTwo": {
+					"label": "Change Extruder 2 Temp",
+					"description": "Select if Second Extruder Temperature has to be changed",
+					"type": "bool",
+					"default_value": false
+				},
+				"i4_extruderTwo": {
+					"label": "Extruder 2 Temp",
+					"description": "New Second Extruder Temperature",
+					"unit": "C",
+					"type": "float",
+					"default_value": 190,
+					"minimum_value": "0",
+					"minimum_value_warning": "160",
+					"maximum_value_warning": "250",
+					"enabled": "i3_Change_extruderTwo"
+				},
+				"j1_Change_fanSpeed": {
+					"label": "Change Fan Speed",
+					"description": "Select if Fan Speed has to be changed",
+					"type": "bool",
+					"default_value": false
+				},
+				"j2_fanSpeed": {
+					"label": "Fan Speed",
+					"description": "New Fan Speed (0-100)",
+					"unit": "%",
+					"type": "int",
+					"default_value": 100,
+					"minimum_value": "0",
+					"minimum_value_warning": "0",
+					"maximum_value_warning": "100",
+					"enabled": "j1_Change_fanSpeed"
+				},
+				"caz_change_retractfeedrate": {
+					"label": "Change Retract Feed Rate",
+					"description": "Changes the retraction feed rate during print (M207)",
+					"type": "bool",
+					"default_value": false
+				},
+				"caz_retractfeedrate": {
+					"label": "Retract Feed Rate",
+					"description": "New Retract Feed Rate (units/s)",
+					"unit": "units/s",
+					"type": "int",
+					"default_value": 40,
+					"minimum_value": "0",
+					"minimum_value_warning": "0",
+					"maximum_value_warning": "100",
+					"enabled": "caz_change_retractfeedrate"
+				},
+				"caz_change_retractlength": {
+					"label": "Change Retract Length",
+					"description": "Changes the retraction length during print (M207)",
+					"type": "bool",
+					"default_value": false
+				},
+				"caz_retractlength": {
+					"label": "Retract Length",
+					"description": "New Retract Length (units)",
+					"unit": "units",
+					"type": "int",
```

Comment:
type should be float not int

Reply:
What about the feedrate? Float as well?

Reply:
I found a few more as well.

Reply:
Done. Have a look and let me know if you need anything else!